sdk/cdn/Azure.ResourceManager.Cdn/src/autorest.md

```diff
@@ -217,19 +217,21 @@ directive:
       for (var key in $) {
```

Comment:
Is this rule can be replaced by `rename-rules` ?

Reply:
Fixed